.github/workflows/workflow.yml

```diff
@@ -0,0 +1,15 @@
+name: Lighthouse Audit
+on: [push, pull_request]
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Audit Netlify deploy preview
+      uses: jakejarvis/lighthouse-action@master
+      with:
+        netlify_site: 'develop--lunieio.netlify.com/'
```

Comment:
@faboweb i can't access netlify - we need a different URL 

https://github.com/marketplace/actions/lighthouse-audit#pull-request-audits-with-netlify-deploy-preview

Reply:
```suggestion
        netlify_site: 'lunieio.netlify.com'
```

Reply:
> i can't access netlify

let's fix that